db/seeds.d/16-mail_notifications.rb

```diff
@@ -37,23 +37,31 @@
     :queryable          => true
   },
 
-  {:name               => :host_built,
-   :description        => N_('A notification when a host finishes building'),
-   :mailer             => 'HostMailer',
-   :method             => 'host_built',
-   :subscription_type  => 'alert'
+  {
+    :name               => 'host_built',
+    :description        => N_('A notification when a host finishes building'),
+    :mailer             => 'HostMailer',
+    :method             => 'host_built',
+    :subscription_type  => 'alert'
   },
 
   {
-    :name               => :tester,
+    :name               => 'tester',
     :description        => N_('A test message to check the email configuration is working'),
     :mailer             => 'UserMailer',
     :method             => 'tester',
-    :subscription_type  => false
+    :subscriptable      => false,
+    :subscription_type  => nil
   }
 ]
 
 notifications.each do |notification|
-  MailNotification.create(notification) if MailNotification.where(:name => notification[:name]).blank?
+  if (mail = MailNotification.find_by_name(notification[:name]))
+    mail.attributes = notification
+    puts mail.changed?
+    mail.save! if mail.changed?
+  else
+    m = MailNotification.create(notification)
+    raise "Unable to create MailNotification: #{format_errors m}" if m.nil? || m.errors.any?
```

Comment:
`raise ::Foreman::Exception.new(N_("Unable to create mail notification: %s"), format_errors(created_mail_notification))`, that way we get an [ERF](http://projects.theforeman.org/projects/foreman/wiki/ErrorCodes) and it's easier to track / write a solution for common problems, etc..

Secondarily, `m` is a bit confusing when you just mean `created_notification` or something like that.